README.md

```diff
@@ -20,29 +19,22 @@ Convene secure, affordable, always-available digital Spaces where anyone can
 
 ## Overview
 
-Convene serves as the _Means of Connection_. Create a Space to co-conspire,
-relax, learn, play, work, swap recipes, read books, or really anything!
+Convene is an _Operating System_ for **Community Owned Digital Infrastructure**. Once installed
+to the cloud, you access Convene via a web browser or on-device apps.
```

Comment:
s/Once installed to cloud/Once you start the server/

If the audience are computer-y people, it seems like they would understand what "starting the server" means.

Reply:
fixed in https://github.com/zinc-collective/convene/pull/749